e2e/pages/adf/filePreviewPage.ts

```diff
@@ -257,10 +256,9 @@ export class FilePreviewPage {
         });
     }
 
-    /*
     zoomOut() {
```

Comment:
I cannot find any place where this method is being called... The expect inside it is not reached because of that. The same case is for `zoomIn` and `actualSize` methods.